src/sentry/sentry_apps/api/endpoints/sentry_app_installations.py

```diff
@@ -90,6 +92,9 @@ def post(self, request: Request, organization) -> Response:
                 sentry_app__slug=slug, organization_id=organization.id
             )
         except SentryAppInstallation.DoesNotExist:
+            assert isinstance(request.user, User) or isinstance(
+                request.user, RpcUser
+            ), "user must be authenticated to create a SentryAppInstallation"
```

Comment:
```suggestion
            assert isinstance(request.user, (User, RpcUser)), "user must be authenticated to create a SentryAppInstallation"
```

You can provide a list of types to isinstance() and it will return True if any option is found. 🏌️

Reply:
🤯🤯🤯